### PR TITLE
Submitting 4/24 HW

### PR DESCRIPTION
**Pull request by teknoformula**

### CODE REVIEW

**Comment by teknoformula** [comment]
submission for 1/28 HW

**Comment by teknoformula** [comment]
initial monument code. to be refactored to meet GA Rubric Standards..

**Comment by teknoformula** [comment]
Submission for 4/30

**Comment by teknoformula** [comment]
Add 1/31 HW:
To-do:
refactor JS and add landing page

**Comment by teknoformula** [comment]
Feb 6 HW

**Comment by teknoformula** [comment]
geoquake lab submission

**Comment by teknoformula** [comment]
6

**Comment by teknoformula** [comment]
geoquake lab submission Feb 6 HW

File: jason-teknoformula/week-1/monument-mockup-master/pt-2/starter-code/css/style.css [hunk 1]
```diff
@@ -0,0 +1,241 @@
+/* ----------------------------------------
+To-Dos
+[x] - sticky header
+[x] - use flexbox for a 2 column layout
+[ ] - add a contact form
+[ ] - add a hover transition on a button
+[ ] - review font-awesome
+------------------------------------------- */
+
```
Comment (DaltonHart) [inline jason-teknoformula/week-1/monument-mockup-master/pt-2/starter-code/css/style.css]: Love this sudo code todo

**Pull request merged**